src/press/models.py

```diff
@@ -108,6 +109,26 @@ def journals():
     def users():
         return core_models.Account.objects.all()
 
+    def site_url(self, path=""):
```

Comment:
Press already has `site_url` from `AbstractSiteModel`

Reply:
Ok. Shouldn’t Journal use this also?